docs/user/videos-and-tutorials/guides/migrating-from-avado.md

```diff
@@ -0,0 +1,251 @@
+
+### Additional Steps To Troubleshoot Your Installation
+
+:::info INFO
+These are additional steps to be followed in case you cannot find the Dappnode wifi after following the [above instructions](./migrating-from-avado.md#how-to-migrate-your-avado-node-to-a-dappnode).
+
+**Note:** If you can see the wifi network "DappNodeWifi" on your computer's list of available networks, you do not have to complete these additional steps.
+:::
+
+To login to your new Dappnode, you will need either:
+
+- An external monitor, external keyboard & HDMI cable.
+- Or, the [internal IP of your connected Avado/Dappnode](./migrating-from-avado.md#how-to-find-your-internal-ip-address).
+
+If you have the internal IP address of your device you can [login using SSH](./migrating-from-avado.md#login-using-ssh), otherwise you can [use an external monitor and keyboard](./migrating-from-avado.md#login-using-an-external-monitor-and-keyboard) to interact with your Avado/Dappnode directly.  
+
+### Login Using SSH
+
+To complete the installation using SSH, you will need to make sure you know your device's internal IP address. You can follow [these steps](./migrating-from-avado.md#how-to-find-your-internal-ip-address) to find your internal IP address if you haven't already.
+
+(**1**) Connect to your now converted dappnode by entering the following command into your terminal:
+
+```bash 
+ssh dappnode@[INSERT_YourInternalIPAddress]
+```
+
+**Note:** replace `[INSERT_YourInternalIPAddress]` with your devices' internal IP address E.g. `192.xxx.x.xxx`.
+
+![Dappnode Login](/img/Dappnode_ssh.png)
+
+(**2**) You will then be asked to log in. Use the following default credentials to log in:
+
+```bash
+Username: dappnode
+Password: dappnode.s0
+```
+
+(**3**) Once you've logged in, complete the installation through the terminal as documented [here.](./migrating-from-avado.md#complete-installation)
+
+### Login Using An External Monitor and Keyboard
+
+You can log in to your node using an external monitor & keyboard. 
+
+(**1**) Plug in your monitor to your Avado/Dappnode using an HDMI cable.
+
+(**2**) Plug in your external keyboard to your Avado/Dappnode device. 
+
+Your monitor should display a screen asking for your login details.
+
+(**3**) Use the following default credentials to log in:
+
+```bash
+Username: dappnode
+Password: dappnode.s0
+```
+
+(**4**) Once you've logged in, complete the installation through the terminal as documented [here.](./migrating-from-avado.md#complete-installation)
+
+### Complete Installation 
+
+These are additional steps to complete the installation of Dappnode software in case you cannot find the Dappnode wifi after following the [above instructions](./migrating-from-avado.md#how-to-migrate-your-avado-node-to-a-dappnode).
+
+Follow these steps after having logged into your now semi-transitioned Avado device. You should see the following screen:
+
+![Dappnode Screen](/img/Dappnode_screen.png)
```

Comment:
This image wasn't added in `/static/img`

`Error: Image static/img/Dappnode_screen.png used in docs/user/videos-and-tutorials/guides/migrating-from-avado.md not found.`

Reply:
fixing now

Reply:
@Marketen hopefully should be good now! https://github.com/dappnode/DAppNodeDocs/pull/423/commits/01307867c931f2779dd45ab865176376bdfc9bf6

Reply:
Seems like it! Thanks for this great guide